app/webpacker/controllers/toggle_controller.js

```diff
@@ -4,9 +4,22 @@ export default class extends Controller {
   static targets = ["content"];
 
   toggle(event) {
+    event.stopImmediatePropagation()
     const input = event.currentTarget;
+    const chevron = input.querySelector(".icon-chevron-down, .icon-chevron-up")
```

Comment:
Side note: any time I need to rummage around for an element in the DOM using `querySelector()` inside a StimulusJS controller, I would stop and think about looking for an alternative, and maybe make the element a named target. There's isn't always a better alternative, and it's not always the best way to go, but I would pause to consider it.